onnxruntime/core/providers/xnnpack/detail/utils.h

```diff
@@ -56,6 +57,50 @@ enum class QuantizedOpType : uint8_t {
   Unknown,
 };
 
+class InternalDataInitializer {
+ public:
+  InternalDataInitializer(const ONNX_NAMESPACE::TensorProto& tensor_proto);
+
+  bool HasExternalData() {
+    return has_external_data_;
+  }
```

Comment:
What's the use-case for a user caring about this? i.e. once they create an instance of the class from the original data, when do they care about where that data was originally located?

Reply:
Xnnpack expected that data is not in external localtion, is that right?